Build failure

With the latest code up to commit 52e06c4, when I run `ipdk build --no-cache` I see the following obtuse error and it fails:

```
-- Build files have been written to: /root/P4OVS_DEPS_SRC_CODE/gflags/build
make: *** empty string invalid as file name.  Stop.
The command '/bin/bash -c /root/start_p4ovs.sh /root &&      /root/scripts/run_cleanup.sh /root ${KEEP_SOURCE_CODE}' returned a non-zero code: 2
/git/ipdk/build
vagrant@ubuntu2004:/git/ipdk/build$
```

--- a/build/scripts/os_ver_details.sh
+++ b/build/scripts/os_ver_details.sh
@@ -60,10 +60,10 @@ get_num_cores()
            NUM_THREADS=$((NUM_CORES / 4))
            NUM_THREADS=-j$NUM_THREADS
        else
-           NUM_THREADS=${NUM_CORES}
+           NUM_THREADS=-j${NUM_CORES}
        fi
     else
         NUM_CORES=1
-        NUM_THREADS=1
+        NUM_THREADS=-j1
     fi
 }

